tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host: Installer host
+    """
+    timeout, interval = 20, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(host).ceph.orch.ls(service_type="mon", format="json")
+        configs = loads(out)
+        for config in configs:
+            running = config.get("status").get("running")
+            size = config.get("status").get("size")
+        if running == size:
+            log.info(f"MON service: size: {size}, running: {running}")
+            return True
+        if w.expired:
+            raise ClusterViewError("MON service check Failed")
+
+
+def verify_mon_ips(host):
+    """
+    Verify MON IPs updated in ceph.conf
+    Args:
+        host: Installer host
+    """
+    # Get updated MON nodes
+    cmd = "cephadm shell ceph orch ps --daemon-type mon --format json"
+    out = host.exec_command(sudo=True, cmd=cmd)[0]
+    configs = loads(out)
+    mon_nodes = [config.get("daemon_id") for config in configs]
+
+    # Get MON node IPs
+    mon_ips = []
+    out = CephAdm(host).ceph.orch.host.ls(format="json")
+    configs = loads(out)
+    for config in configs:
+        if config.get("hostname") in mon_nodes:
+            mon_ips.append(config.get("addr"))
+
+    # Validate MON IPs updated in ceph.conf
+    mon_ips_conf = host.remote_file(file_name="/etc/ceph/ceph.conf", file_mode="r")
```

Comment:
For fix configs, create separate global variable.

Reply:
Updated the global variable.